### ISSUE TITLE
about the performance

### ISSUE THREAD

**Issue by kbyran** [justayak/cabbage#9]
I had tested on the MOT16, but got very poor performance. 
Is anyone successful on reproducing the numbers on paper? 
Maybe we can discuss on improving the project and help each other. @jutanke

**Comment by shensheng27** [comment]
@kbyran I am trying to test this repo on MOT16. However the code is very time costing. I using 14k detection and dmax=100 to test on video MOT16-02, but the c++ graph solver take days to solve 30million edge graph and no result for now.
I wonder did you test on all MOT16 test series or just MOT16 train or just MOT16-02? How long for you to run test. 
Thanks

**Comment by kbyran** [comment]
@shensheng27 yes, it’s time costing. Deleting some detections with low confidence could help.

**Comment by stasysp** [comment]
@kbyran  we have the same problem, @jutanke could you share please the pretrained models?

**Comment by derkbreeze** [comment]
I think setting the maximum temporal gap between nodes to a small value, e.g. 10 frames in the original paper should help. Multicut itself is NP-hard, meaning the bigger the graph size, the much more time it requires to solve.
